boot/mynewt/src/main.c

```diff
@@ -46,6 +46,12 @@
 #include "bootutil/bootutil.h"
 #include "bootutil/bootutil_log.h"
 
+#if MYNEWT_VAL(BOOT_CUSTOM_START)
+static struct boot_rsp rsp;
+static uintptr_t flash_base;
+void boot_custom_start(uintptr_t flash_base, struct boot_rsp *rsp);
+#endif
```

Comment:
i don't understand why we need these.  The function call below can just use the flash_base and rsp we have already.